tests/ui/packed/packed-array-const-generic.rs

```diff
@@ -0,0 +1,19 @@
+//@ check-pass
+#![allow(dead_code)]
+
+#[repr(C, packed)]
+struct PascalString<const CAP: usize> {
+    len: u8,
+    buf: [u8; CAP],
+}
+
+fn bar<const CAP: usize>(s: &PascalString<CAP>) -> &str {
+    // 目标：这行不应触发 E0793
```

Comment:
I'm just the bug reporter, but I would guess that only English comments will be accepted into rustc, probably.